.github/workflows/shared_meterpreter_acceptance.yml

```diff
@@ -159,7 +159,7 @@ jobs:
       - name: Get mettle version
         if: ${{ matrix.meterpreter.name == 'mettle' && inputs.build_mettle }}
         run: |
-          echo "METTLE_VERSION=$(grep -oh '[0-9].[0-9].[0-9]*' lib/metasploit_payloads/mettle/version.rb)" | tee -a $GITHUB_ENV
+          echo "METTLE_VERSION=$(ruby -ne 'puts $& if /[0-9].[0-9].[0-9]*/' lib/metasploit_payloads/mettle/version.rb)" | tee -a $GITHUB_ENV
```

Comment:
```suggestion
          echo "METTLE_VERSION=$(ruby -ne 'puts $& if /[0-9][.][0-9][.][0-9]*/' lib/metasploit_payloads/mettle/version.rb)" | tee -a $GITHUB_ENV
```

Otherwise, `.` will match any character.

Reply:
@cgranleese-r7 It'd be worth verifying that the old/new regexes supports multiple digit releases too, i.e. `1.10.0`

Reply:
Hopefully this link works - https://regex101.com/r/W8QwSG/1

I looked at this again this morning and @adfoster-r7 was right, it wasn't matching `1.10.0`. So I have landed on the following regex, which seems to cover everything I need 🤞:
```
\d+\.?(\*|\d+\.)?(\*|\d+)
```
This will match:
- 1.1
- 11.1
- 1.11
- 11.11
- 111.111.1
- 111.111.111
- 1.0.0
- 1.1.0